pkg/engine/s3.go

```diff
@@ -46,6 +46,10 @@ func (e *Engine) ScanS3(ctx context.Context, c sources.Config) error {
 	}
 
 	s3Source := s3.Source{}
+	ctx = context.WithValues(ctx,
+		"source_type", s3Source.Type(),
+		"name", "trufflehog - gitlab",
```

Comment:
This should be S3

Reply:
would it be better to say `<name> source`?

Reply:
Ah, nice catch!

> would it be better to say `<name> source`?

I was just copying the name that we pass to the source, but I think either one is good. Would that be `s3 source` in this case?

Reply:
I decided to make the key `"source_name"` and the value just `"s3"`.